packages/app-accounts/src/index.css

```diff
@@ -2,6 +2,6 @@
 /* This software may be modified and distributed under the terms
 /* of the ISC license. See the LICENSE file for details. */
 
-.accounts--Modal-Address {
+.accounts--ChangePass-Modal-Address {
```

Comment:
I don't think this is correct - it is shared, i.e. 

- ChangePass
- Forgetting
- Backup